src/components/site/Images.md

```diff
@@ -0,0 +1,22 @@
+```jsx
+<Images
+  imageSrc="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+  content="Lorem Ipsum is simply dummy text of the printing and typesetting of Lorem Ipsum.industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five"
+></Images>
+```
+
+Html Snippet:
+```html
+<div>
+<div class="dg_image">
```

Comment:
the dg_image class shouldn't be on a div as the Site Executive users won't be creating a div that wraps around an image when they are using the cms.

Reply:
in this case, we want default functionality here, so we probably don't even need to add a class to this image, and rather do something like this in the css

```scss
#main-content img
```

this targets images inside of the content section of the page, and won't impact the images on the rest of the site.